UPGRADING.md

```diff
@@ -1,6 +1,19 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.5.1
+
+#### Dependent params
+
+If you use [dependent params](https://github.com/ruby-grape/grape#dependent-parameters) with
+`Grape::Extensions::Hash::ParamBuilder`, make sure a parameter to be dependent on is set as a Symbol
```

Comment:
Explain what won't work? In this case a `String`? Also add a period at the end of the sentence or maybe a before/after section?

Reply:
> Also add a period at the end of the sentence or maybe a before/after section?

I am not sure what you mean here :blush: Could you explain?

Reply:
a Symbol. (it's a full sentence, just missing a period)

Reply:
Regarding before/after, you're saying make sure it's a Symbol. What happens if it's not? Explain that.